client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationMetadata/InvestigationMetadata.tsx

```diff
@@ -1,27 +1,44 @@
 import React from 'react';
 import {Paper} from '@material-ui/core';
+import { format } from 'date-fns';
+
+import { InvestigationMetaData } from 'models/InvestigationInfo';
+
 import InfoItem from '../InfoItem';
 import useStyles from './InvestigationMetadataStyles';
 
-const InvestigationMetadata = () => {
+const InvestigationMetadata = (props: Props) => {
+
     const classes = useStyles();
-    const investigationStartDate = '26/08/2020';
-    const lastUpdateTime = '26/08/2020';
-    const district = 'דרום';
-    const investigator = 'אברהם כהן';
-    const updatingInvestigator = 'פלוני זהבי';
-    const investigatorPhoneNumber = '054-1234567';
+    const { investigationMetaData } = props;
 
     return (
         <Paper className={classes.metadata}>
-            <InfoItem name='תאריך תחילת החקירה' value={investigationStartDate} />
-            <InfoItem name='תאריך עדכון אחרון' value={lastUpdateTime} />
-            <InfoItem name='נפח/מחוז' value={district} />
-            <InfoItem name='מבצע החקירה' value={investigator} />
-            <InfoItem name='משתמש מעדכן' value={updatingInvestigator} />
-            <InfoItem name='טלפון המבצע' value={investigatorPhoneNumber} />
+            <InfoItem name='תאריך תחילת החקירה' value={
+                    format(new Date(investigationMetaData.startTime), 'dd/MM/yyyy')
+                }
+            />
+            <InfoItem name='תאריך עדכון אחרון' value={
+                    format(new Date(investigationMetaData.lastUpdateTime), 'dd/MM/yyyy')
+                }
+            />
+            <InfoItem name='נפח/מחוז' value={investigationMetaData.investigatingUnit} />
+            <InfoItem name='מבצע החקירה' value={
+                    investigationMetaData.userByCreator.personByPersonId.firstName + ' '
+                    + investigationMetaData.userByCreator.personByPersonId.lastName
+                }
+            />
+            <InfoItem name='משתמש מעדכן' value={
+                    investigationMetaData.userByLastUpdator.personByPersonId.firstName + ' '
+                    + investigationMetaData.userByLastUpdator.personByPersonId.lastName
```

Comment:
that formatting can be generated by function the receives a person type

Reply:
good idea